task03-one-piece-website/index.html

```diff
@@ -329,5 +329,13 @@ <h3 class="mistObject__card__name">Gomu Gomu No Mi</h3>
         </a>
       </div>
     </div>
+
+    <div class="footer">
```

Comment:
Be aware of the different between section and div https://www.geeksforgeeks.org/what-is-the-difference-between-section-and-div-tags-in-html/#:~:text=Both%20the%20tags%20(,is%20called%20a%20division%20tag.